incubator/mail2most/templates/pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if .Values.persistence.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "mail2most.fullname" . }}-pvc
+  labels:
+    app: {{ template "mail2most.fullname" . }}
```

Comment:
use the new labels, see: https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md#standard-labels